app/uart.h

```
@@ -303,40 +303,81 @@ void uart_event_handle(app_uart_evt_t* p_event)
+                                    {
+                                        err_code = sd_ble_gap_auth_key_reply(m_conn_handle, BLE_GAP_AUTH_KEY_TYPE_NONE, NULL);
+                                        APP_ERROR_CHECK(err_code);
+                                        err_code = sd_ble_gap_disconnect(m_conn_handle, BLE_HCI_REMOTE_USER_TERMINATED_CONNECTION);
+                                        APP_ERROR_CHECK(err_code);
+                                    }
+                                    APP_ERROR_CHECK(err_code);
+                                    waiting_passkey_response = false;
+                                }
+                                break;
+                            case BLE_PASSKEY_REJECT:
+                                if(waiting_passkey_response && m_conn_handle != BLE_CONN_HANDLE_INVALID)
```

Clang (14.0.6)

[note] +5, including nesting penalty of 4, nesting level increased to 5

(clang)